toonz/sources/common/tgl/tgl.cpp

```diff
@@ -11,12 +11,14 @@
 #ifndef __sgi
 #ifdef WIN32
 #include <GL/glut.h>
+#elif defined(LINUX)
```

Comment:
Noticed this, but Win32 and Linux include blocks will likely diverge, so don't think its worth it to de-duplicate a single include.